lib/rex/proto/mssql/client.rb

```diff
@@ -640,11 +640,11 @@ def powershell_upload_exec(exe, debug=false)
           print_status("Be sure to cleanup #{var_payload}.exe...")
         end
 
-        def address
+        def peeraddress
```

Comment:
changing these names but not updating any calls seems a bit sus, if these were being used before then any calls would need updated too, if they weren't being used before was that a bug? are they being used now with this rename?